estate_account/__manifest__.py

```diff
@@ -0,0 +1,7 @@
+{
+    'name': 'Real Estate Account',
+    'depends': ['base', 'estate', 'account'],
```

Comment:
Avoid redundancies here as `estate` already relies on `base`
```suggestion
    'depends': ['estate', 'account'],
```